src/components/UsageBar.tsx

```diff
@@ -0,0 +1,93 @@
+'use client'
+
+import { ReactNode, useEffect, useState } from 'react'
+import { useKeyring, Plan } from '@w3ui/react-keyring'
+
+const B = 1024
+const MB = 1024 * B
+const GB = 1024 * MB
+const BarHeight = 10
+
+const PlanLimit: Record<string, number> = {
+  'did:web:free.web3.storage': 5 * GB,
+  'did:web:pro.web3.storage': 25 * GB,
+  'did:web:enterprise.web3.storage': 100 * GB
+}
```

Comment:
these are listed as 5GB, 100GB and 2TB in the pricing page on the website and in stripe. @dchoi27 can you confirm these thresholds?